airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1331,6 +1331,23 @@ definitions:
       $parameters:
         type: object
         additionalProperties: true
+  ResponseToFileExtractor:
+    title: CSV To File Extractor
+    description: Record extractor that downloads a headless
```

Comment:
What is `headless`? My understanding of this component is that we need to use it when the responses could be too big in memory and we need to save them on disk to avoid OOM issues. Can we add something in the description to reflect that?

Reply:
description updated